test/testModuleUsage.js

```diff
@@ -105,22 +105,24 @@ describe('Using ERC-Checker as node-module', function () {
 		});
 
 		describe('for two differing papers containing 2 images each', function () {
-			it('should contain no errors, and a parameter {"differencesFound" == true}, plus an Array of image comparison results with a length 2', function (done) {
+			it('should contain no errors, and a parameter {"checkSuccessful" == false}, i.e. differences exist, plus an Array of image comparison results with a length 2', function (done) {
 				this.timeout(0);
-				checker(testStringA, testStringB, null, "test_2Img_OnlySecondDiffering")
+				checker(testStringA, testStringB, "test_2Img_OnlySecondDiffering")
 					.then(
 						function (metadata) {
-							if (metadata.differencesFound == true
+							if (metadata.checkSuccessful == false
```

Comment:
all these different checks in one ìf` clause should be independent assertions. If _one_ of these breaks, then you will have to dig into the test to find out which one.

**This is the case in a few other tests, too!**